install/gcp/installer/forseti_server_installer.py

```diff
@@ -200,6 +206,17 @@ def get_configuration_values(self):
                 the forseti configuration file.
         """
         bucket_name = self.generate_bucket_name()
+
+        resource_root_id = ''
+        composite_root_resources = ''
+
+        if self.composite_root_resources:
```

Comment:
should this logic go in the jinja template?

Reply:
We're currently building the initial server config yaml file here.